tool/resources/org/antlr/v4/tool/templates/codegen/Go/Go.stg

```diff
@@ -746,8 +798,9 @@ InvokeRule(r, argExprsChunks) ::= <<
 	var _x = p.<r.name; format="cap">(<argExprsChunks>)
 	<endif>
 
+    <r.labels:{l | <labelRefStuff(l,"_x")> }; separator="\n">
```

Comment:
Does `go fmt` run on code that's auto-generated by `go generate`?

Anyway, right now I'm committing the generated code because I didn't know about `go generate`. 👍